PocketKit/Sources/Analytics/AppEvents/Search.swift

```diff
@@ -289,9 +289,11 @@ public extension Events.Search {
     static func searchCardContentOpen(
         url: URL,
         positionInList: Int,
-        scope: SearchScope
+        scope: SearchScope,
+        destination: ContentOpen.Destination
     ) -> ContentOpen {
         return ContentOpen(
+            destination: destination,
```

Comment:
when should we be including destination in our Content Opens? Looks like it's differentiating web view vs. reader view

Reply:
cc @kirill-demtchouk can you advise?

Reply:
`destination` is a required field on the `content_open` event, so we should always include it. Can we set it to `internal` if the item opens in Article view and `external` if the item opens in Web view? Otherwise we can always set it to `internal` given that the mobile apps never hand off the user to an external app when they open content.

Reply:
We can do that.